### ISSUE TITLE
Running Multiple Instances of Chatty

### ISSUE THREAD

**Issue by gamingneeds** [chatty/chatty#179]
I've ran multiple instances of Chatty with success by cloning the folder multiple times. The only issue is that when I open multiple instances for the next use, it only loads the last username I used, and I'm forced to re-auth the rest of the instances.

Anyway to have them save information in separate profiles, or just add native support for multiple instances in a separate version?

=============================

On a separate note, I've used Chatty to successfully earn "loyalty" points for viewing a stream by having the stream open in Chatty. However, if a stream offers in-game drops for viewing the stream, Chatty doesn't seem to trigger that I was there. Is there something else they use to enable in-game drops for viewership, as I assumed it was always just the list of viewers?

Thanks! :)

**Comment by tduva** [comment]
http://chatty.github.io/help/help-guide1.html

I don't know what Twitch is using for Drops, possibly the precense system that tracks what you are doing on the website.

**Comment by gamingneeds** [comment]
I know I can open a bunch of browser windows with different logins and the items will drop, when I'm inactive and have them minimized, so not sure what it could be. :/

Thanks for the help link.

**Comment by gamingneeds** [comment]
I found this link which seems to shed some light on the way drops are delivered. From what I could read, I don't see anything that would read differently?

https://dev.twitch.tv/docs/drops

**Comment by gamingneeds** [comment]
Hey tduva, did you ever have a chance to look at that article? It'd be amazing if you could make it where Chatty was able to trigger that API.

Thanks!

**Comment by tduva** [comment]
That is for game developers that want you give viewers drops, not an API for viewers to receive drops.

**Comment by gamingneeds** [comment]
Yeah, I understand that, I figured the documentation would help to understand how the API works, to see if it was possible for Chatty to eventually "trigger" whatever it's currently not triggering. If I use Chatty versus a normal browser to participate in a stream, at this point it fails to count me as an active viewer.

**Comment by craftwar** [comment]
Currently you can start Chatty with/without -cd to use different login data.
I think add an option to specify profile directory will be better.
In -cd case, there are the same caches in two copies.

**Comment by tduva** [comment]
You can specify the working directory in the shortcut you create, which will be the settings directory.

**Comment by craftwar** [comment]
I start chatty in cmd script.

start /D"D:\Chatty" javaw -da -Xbatch -Xmx200M -jar "D:\Chatty\Chatty.jar" -cd blah
or 
start /D"D:\Chatty" javaw -da -Xbatch -Xmx200M -jar "D:\Chatty\Chatty.jar" -cd "D:\Chatty\blah"

Both are not working in my case.

only -cd work

**Comment by tduva** [comment]
The working directory is associated with the process. In your command you specify it with the `/D` parameter.

**Comment by craftwar** [comment]
In my experience, changing working directory to not program root directory breaks resource path.
User cares about where profile is saved not where working directory is. (I don't think user doesn't program knows what it is.)
In my opinion, give user a profile directory is easier to understand.

start /D"D:\Chatty" javaw -da -Xbatch -Xmx200M -jar "D:\Chatty\Chatty.jar"
profile in %appdata%/.chatty (not working directory)
start /D"D:\Chatty" javaw -da -Xbatch -Xmx200M -jar "D:\Chatty\Chatty.jar" -cd
chatty root

**Comment by tduva** [comment]
Changing the working directory works fine for Chatty, I'm using it everyday like that. However I do agree that it's a convoluted way of specifying the settings directory, which is why I just added a new `-d` parameter for it.

**Comment by craftwar** [comment]
"-d" is not in document, need fix

**Comment by tduva** [comment]
I *just* added it, meaning it's not in any released version yet. o_O
